README.md

```diff
@@ -259,7 +259,65 @@ BundleKeys.endpoints := Map("akka-remote" -> Endpoint("tcp"))
 
 In the above, no declaration of `services` is required as akka remoting is an internal, cluster-wide TCP service.
 
-## play[23|24|25]-conductr-bundle-lib
+## play25-conductr-bundle-lib
+
+> If you are using Play 2.5 then this section is for you. Otherwise jump below to the "play[23|24]-conductr-bundle-lib" section.
+
+This library provides a reactive API using [Play WS](https://www.playframework.com/documentation/2.5.x/ScalaWS) and should be used when you are using Play. The library depends on `akka24-conductr-bundle-lib` and can be used for both Java and Scala. As per Play's conventions, `play.api` is used for the Scala API and just `play` is used for Java.
+
+As with `conductr-bundle-lib` there are two services:
+
+* `com.typesafe.conductr.bundlelib.play.LocationService` (Java) or `com.typesafe.conductr.bundlelib.play.api.LocationService` (Scala)
+* `com.typesafe.conductr.bundlelib.play.StatusService` (Java) or `com.typesafe.conductr.bundlelib.play.api.StatusService` (Scala)
+
+and there is also another:
+
+* `com.typesafe.conductr.bundlelib.play.Env` (Java) or `com.typesafe.conductr.bundlelib.play.api.Env` (Scala)
+
+Please read the section on `conductr-bundle-lib` and then `scala-conductr-bundle-lib` for an introduction to these services. The `Env` one is discussed in the section below. The major difference between the APIs for Play 2.5 and the other variants is that components are expected to be injected. For example, to use the `ServiceLocator` in your controller (Scala):
+
+```scala
+class MyGreatController @Inject() (serviceLocator: ServiceLocator, locationCache: CacheLike) extends Controller {
+  ...
+  locationService.lookup("/known", URI(""), locationCache)
+  ...
+}
+```
+
+The following components are available for injection:
+
+* CacheLike
+* ConnectionContext
+* ServiceLocator
+* StatusService
+
+Note that your `application.conf` should contain the following (for Scala):
+
+```scala
+play.application.loader = "com.typesafe.conductr.bundlelib.play.lib.ConductRApplicationLoader"
+
+play.modules.enabled ++= Seq(
+  "com.typesafe.conductr.bundlelib.play.api.BundlelibModule",
+  "com.typesafe.conductr.bundlelib.play.api.ConductRLifecycleModule"
+  )
```

Comment:
This is sbt syntax, not HOCON :)

That aside, how often will it be that people won't want these two modules enabled?  If that's rare (or never), then I would recommend that the library add these to `reference.conf`.  The `play.application.loader` is fine to be explicitly configured by the user, since Play already defines a default for it in `reference.conf`, but Play's dynamic module registration is designed to work such that the user shouldn't need to manually enable modules.